modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextStore.java

```diff
@@ -113,6 +114,16 @@ void storeDurableSubscription(AndesSubscription subscription)
     void storeNodeDetails(String nodeID, String data) throws AndesException;
 
     /**
+     * Store cluster events such as QUEUE_ADDED, BINDING_ADDED, SUB_REMOVED, etc in the database.
+     *
+     * @param eventType    A string defining the type of the even. e.g., QUEUE_ADDED
+     * @param eventDetails Event details encoded as a String
+     * @param timestamp    TODO remove this
```

Comment:
Need to fix the parameter comments. :)
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@param